Add documentation changes, analytics metrics. (#544)
mulesoft/docs-runtime-manager: modules/ROOT/pages/anypoint-platform-cli.adoc

`> set HTTPS_PROXY=https://__username__:__password__@__proxy-server__:443`
--

== Notes

> * To improve Anypoint CLI experience, we send anonymized usage. You may opt-out by setting `collectMetrics` to false in the `credentials` file.

Reviewer: ```suggestion
* To improve the Anypoint CLI experience, we gather anonymized usage data. You can opt out by setting `collectMetrics` to false in the `credentials` file.
```
